experimental/PartitionedPermutations/src/PartitionedPermutation.jl

```diff
@@ -0,0 +1,88 @@
+"""
+    PartitionedPermutation
+
+The type of partitioned permutations. Fieldnames are
+- p::Perm{Int} - a permutation
+- V::SetPartition - a partition
+- check::Bool = true
+If the permutation has length `n`, then the partition must have `n` upper points and 0 lower points. 
+Further, if `W` is the partition given by the cycles of `p`, then `W` must be dominated by `V` in the 
+sense that every block of `W` is contained in one block of `V`. There is one inner constructer of PartitionedPermutation:
+- PartitionedPermutation(_p::Perm{Int}, _V::Vector{Int}) constructs the partitioned permutation where the partition is given by the vector _V.
+If the optional flag `check` is set to `false`, then the constructor skips the validation of the requirements mentioned above.
+
+# Examples
+```jldoctest
+julia> PartitionedPermutation(Perm([2, 1, 3]), [1, 1, 2])
+PartitionedPermutation((1,2), SetPartition([1, 1, 2], Int64[]))
+```
+"""
+struct PartitionedPermutation
+    p::Perm{Int}
+    V::SetPartition
+    check::Bool
+
+    function PartitionedPermutation(_p::Perm{Int}, _V::Vector{Int}, _check::Bool=true)
```

Comment:
```suggestion
    function PartitionedPermutation(_p::Perm{Int}, _V::Vector{Int}; check::Bool=true)
```
`check` should be a kwarg.